x/posts/internal/types/post_test.go

```diff
@@ -760,6 +760,93 @@ func TestPost_Equals(t *testing.T) {
 	}
 }
 
+func TestPost_Unique(t *testing.T) {
```

Comment:
We don't need this as it's going to be tested inside `TestPost_GetPostHashtags` as well. It's fine to delete it and shouldn't change the test coverage.